tests/unit/types/arrays/test_neural_ops.py

```diff
@@ -425,3 +425,16 @@ def test_dam_get_embeddings(tmpdir):
     np.testing.assert_almost_equal(
         da.get_attributes('embedding')[10:20], da._get_embeddings(slice(10, 20))
     )
+
+
+def test_embeddings_setter():
+    emb = np.random.random((100, 128))
+    da = DocumentArray([Document() for _ in range(100)])
+    da.embeddings = emb
+    np.testing.assert_almost_equal(da.embeddings, emb)
```

Comment:
can u also assert that one by one each document has the right embedding vector?

Reply:
I would say both statements are basically the same, right? since to get .embeddings  we iterate over each document and get the embeddig vector

Reply:
yes, but I want to specifically check that accessing each of them gives same result

Reply:
updated